lib/minesweeper_2pl/boardcli.rb

```diff
@@ -0,0 +1,150 @@
+module Minesweeper
+  class BoardCli
+
+    attr_accessor :message, :show_bombs
```

Comment:
The `message` attribute looks unused to me. There are automated tools called "linters" that will read your code and catch style/formatting errors like this. Many of them will even integrate with your editor to show errors in real time, and some of them will even automatically reformat your code. For Ruby, I would look at RuboCop: https://rubocop.readthedocs.io/en/latest/